lib/services/authenticate.dart

```diff
@@ -182,4 +185,16 @@ class FireStoreUtils {
     await auth.FirebaseAuth.instance
         .sendPasswordResetEmail(email: emailAddress);
   }
+
+  static String? calculateTenantId(String? tenantId) {
+    log.logger.d("Calculating tenant ID (with the input of: $tenantId)");
+    String? result = tenantId;
+    if (isNullOrEmpty(result)) {
+      log.logger.d(
+          "Input tenant ID value was null, about to check if it is given at environment variable level using the TENANT_ID variable key.");
+      result = const String.fromEnvironment("TENANT_ID");
```

Comment:
What sets the 'TENANT_ID' env variable?

Reply:
right now the one who builds the application. later on, it can be abstracted but this way, I wanted to hide the kind of tenant values that we're going to use (we'll discuss it during one of the syncs/meetings)